utils/run.sh

```diff
@@ -6,12 +6,19 @@ VIRTUALENV="$FAME_ROOT/env"
 
 if [ -d "$VIRTUALENV" ]; then
     echo "[+] Using existing virtualenv."
+    . "$VIRTUALENV/bin/activate"
 else
     echo "[+] Creating virtualenv..."
     python3 -m virtualenv -p python3 "$VIRTUALENV" > /dev/null
+    . "$VIRTUALENV/bin/activate"
+    pip3 install -r "$FAME_ROOT"/requirements.txt
```

Comment:
Why this change ? The installation script is already installing the requirements.

Reply:
The idea here is to install python requirements when performing `docker build .`, instead of when launching the container.

In order to do this, I had to 
- update utils/run.sh to install requirements when creating the environment
- and run utils/run.sh in the Dockerfile itself

I still kept the requirements.txt installation in the .py, in order to not disturb non-docker installations

Reply:
non-docker installations are also using `utils/run.sh` so it would probably make more sense to move requirements installation in `run.sh` in all cases.

Reply:
Done